modules/auxiliary/scanner/http/symantec_brightmail_ldapcreds.rb

```diff
@@ -0,0 +1,283 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require "base64"
+require 'digest'
+require "openssl"
+
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Symantec Messaging Gateway 10 LDAP Creds Graber',
+      'Description'    => %q{
+          This module will  grab the AD account saved in Symantec Messaging Gateway and then decipher it using the disclosed symantec pbe key.  Note that authentication is required in order to successfully grab the LDAP credentials, you need at least a read account. Version 10.6.0-7 and earlier are affected
+
+      },
+      'References'     =>
+        [
+          ['URL','https://www.symantec.com/security_response/securityupdates/detail.jsp?fid=security_advisory&pvid=security_advisory&year=&suid=20160418_00'],
+          ['CVE','2016-2203'],
+          ['BID','86137']
+        ],
+
+      'Author'         =>
+        [
+          'Fakhir Karim Reda <karim.fakhir[at]gmail.com>',
+        ],
+       'DefaultOptions' =>
+        {
+          'SSL' => true,
+          'SSLVersion' => 'TLS1',
+          'RPORT' => 443
+        },
+       'License'        => MSF_LICENSE,
+       'DisclosureDate' => "Dec 17 2015"
```

Comment:
This DisclosureDate looks incorrect. According to the given URL, disclosure date should be April 18, 2016.